js/script.js

```diff
@@ -93,17 +93,39 @@ function getRandomQuote () {
   return quotes[getRandomQuote];
 }
 console.log(getRandomQuote());
-
 /***
  * `printQuote` function
 ***/
 function printQuote () {
-  let message = "";
-  let showQuote = getRandomQuote(quotes);
+  let display = getRandomQuote(quotes);
+  let message = `
+    <p class = "quote"> ${display.quote}</p>
+    <p class = "source"> ${display.source}</p>
+    <p class = "citation"> ${display.citation}</p>
+    <p class = "year"> ${display.year}</p>
+    <p class = "tag"> ${display.tag}</p> `
+  
+  if (display.quote) {
+    message += `<span class = "quote"> ${display.quote}</span>`
+  }    
```

Comment:
const { quote, source, citation, year} = getRandomQuote(quotes);  
 const message = 
 <p class = "quote"> ${quote || 'na'}</p>
     <p class = "source"> ${source|| 'na'}</p>
     <p class = "citation"> ${citation|| 'na'}</p>
     <p class = "year"> ${year}</p>;

Reply:
or you could do something like 

```
 const message = `
 <p class = "quote"> ${quote || 'na'}</p>
     ${!!source && `<p class = "source"> ` Source: ${source|}`</p>`}
   ${!!citation && ` <p class = "citation"> `Citation: ${citation}`</p>`} 
   ${!!year && ` <p class = "year"> `Year: ${year }`</p>`} `;
```

Reply:
try those out and then lookup any syntax or anything you don't understand on MDN, and then I can help if you still are confused or want additional info

Reply:
-Two pipes mean "or"
-"na" not applicable